src/libraries/System.Net.Quic/tests/FunctionalTests/QuicStreamConnectedStreamConformanceTests.cs

```diff
@@ -27,6 +27,7 @@ public sealed class MockQuicStreamConformanceTests : QuicStreamConformanceTests
     public sealed class MsQuicQuicStreamConformanceTests : QuicStreamConformanceTests
     {
         protected override QuicImplementationProvider Provider => QuicImplementationProviders.MsQuic;
+        protected override Type UnsupportedConcurrentExceptionType => typeof(InvalidOperationException);
```

Comment:
Isn't this default on `StreamConformanceTests`? Do we need to override it?

Reply:
No, I will remove it